Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -298,6 +302,23 @@ void CCodeView::OnPopupMenu(wxCommandEvent& event)
     Refresh();
     break;
 
+  case IDM_ASSEMBLE:
+  {
+    if (!PowerPC::HostIsRAMAddress(m_selection))
+      break;
+    AssemblerEntryDialog dialog(m_selection, this, _("Enter instruction code:"));
+    if (dialog.ShowModal() == wxID_OK)
+    {
+      unsigned long code;
+      if (dialog.GetValue().ToULong(&code, 0) && code <= std::numeric_limits<u32>::max())
+      {
+        m_debugger->InsertBLR(m_selection, code);
```

Comment:
I know, that's why I created https://github.com/dolphin-emu/dolphin/pull/4676 in first place (and it needs some more polish, nonetheless). But that issue is deeply anchored to the debug interface, unfortunately.